tests/wpt/mozilla/tests/mozilla/htmlanchorelment-search-attribute.html

```diff
@@ -0,0 +1,17 @@
+<!doctype html>
+<meta charset="utf-8">
+<html>
+<head>
+<title>HTMLAnchorElement search setter crashes</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<a>anchor</a>
```

Comment:
to rephrase my question (making me hesitate to just move the test):
Is there a specific location where to put it, before it formally enters the WPT repo?

what you called:
> the directory of upstream tests